ui-tests/src/test/java/org/openmrs/reference/page/RegistrationEditSectionPage.java

```diff
@@ -23,7 +23,7 @@
     private static final By STATE_PROVINCE = By.id("stateProvince");
     private static final By COUNTRY = By.id("country");
     private static final By POSTAL_CODE = By.id("postalCode");
-    private static final String STR = "Must be a valid phone number (with +, -, numbers or parentheses))";
+    private static final String PHONENUMBER_VALIDATOR = "Must be a valid phone number (with +, -, numbers or parentheses))";
```

Comment:
@sherrif10 what is your thought about `PHONENUMBER_VALIDATOR` vs `PHONE_NUMBER_VALIDATOR` vs `TELEPHONE_VALIDATOR` ? Which one would be the best for you?

Reply:
> PHONE_NUMBER_VALIDATOR

Sure this sounds good